fastdom.js

```diff
@@ -120,28 +124,51 @@ function scheduleFlush(fastdom) {
 }
 
 /**
- * Runs queued `read` and `write` tasks.
+ * Schedules a new read
+ * batch if one isn't pending.
+ *
+ * @private
+ */
+
+function scheduleReadFlush(fastdom) {
+  if (fastdom.scheduledRead) return;
+  fastdom.scheduledRead = true;
+  if(fastdom._isWriting){
+    setTimeout(fastdom._flushReads, 0);
+  } else {
+    fastdom.immediate.then(fastdom._flushReads);
+    //in case we use setTimeout or a bad Promise polyfill, setTimeout can be called later than a rAF.
+    //by additionally using the ordinary scheduleFlush function we make sure that reads are called as early as possible.
+    scheduleFlush(fastdom);
```

Comment:
@wilsonpage 
Are you fine with this timeout change? I really would like to merge this into the PR.